### PR TITLE
Batch cropping feedback

### PR DESCRIPTION
**Pull request by Reettaphant**
Adds better feedback for batch cropping. 

- Disable the crop button if all images in selection have full frame crops
- Tooltip explaining why button is disabled
- Crop icon spins when cropping is in progress

screen shot 2019-02-06 at 11 46 19

### CODE REVIEW

**Comment by paperboyo** [comment]
👏 
Lacking screenshots/anigifs, is the spinning the same as the spinning on the cropping page?

**Comment by Reettaphant** [comment]
@paperboyo Sorry, forgot to attach the visual aids. The spinning is the same.

**Comment by paperboyo** [comment]
Thanks! Minor: in the tooltip, I would replace `full crops` with `full frames`.

File: kahuna/public/js/components/gr-batch-export-original-images/gr-batch-export-original-images.js [hunk 1]
```diff
@@ -9,6 +9,16 @@ batchExportOriginalImages.controller('grBatchExportOriginalImagesCtrl', [
     function($scope, $rootScope, $state, mediaCropper) {
         let ctrl = this;
 
+        const checkForFullCrops = () => ctrl.images.every(
+            image => image.data.exports.some(
```
Comment (akash1810) [inline kahuna/public/js/components/gr-batch-export-original-images/gr-batch-export-original-images.js]: Not blocking getting this out as we can raise in a further PR, however, should we push this into the [imageAccessor](https://github.com/guardian/grid/blob/master/kahuna/public/js/services/image-accessor.js) to keep knowledge of the shape of the API in one place?

Comment (Reettaphant) [reply kahuna/public/js/components/gr-batch-export-original-images/gr-batch-export-original-images.js]: Yes we should, I've made a card for it.

**Pull request merged**